scripts/generator_utils.py

```diff
@@ -0,0 +1,211 @@
+import yaml
+from argparse import ArgumentParser
+
+
+def common_args() -> ArgumentParser:
+    """
+    Common argument parser for each script
+    """
+    parser = ArgumentParser()
+    parser.add_argument("ConfigFilePath", metavar='config_file_path', type=str, help="Configuration data file path.")
+    parser.add_argument("--settings", type=str, required=False, help="Settings module")
+    return parser
+
+
+def update_model(model, **kwargs):
+    """
+    Helper function for updaing a django model
+    Args:
+        model: Django Model
+        kwargs: Keyword args for each field values
+    """
+    for attr, val in kwargs.items():
+        setattr(model, attr, val)
+    model.save()
+
+
+def deep_merge(source, destination):
+    """
+    helper function to merge two dictionaries
+    """
+    for key, value in source.items():
+        if isinstance(value, dict):
+            # get node or create one
+            node = destination.setdefault(key, {})
+            deep_merge(value, node)
+        else:
+            destination[key] = value
+
+    return destination
+
+
+class Config:
+    """
+    Configuration generation helper class
+    """
+
+    # key value map of microsites
```

Comment:
Map itself suggests that it has key and value,  and this comment seems redundant, don't you think? :)

Reply:
Removed